client/components/App.jsx

```diff
@@ -5,16 +5,18 @@ import { Container } from 'semantic-ui-react'
 import FoodList from './FoodList'
 import TopMenu from './TopMenu'
 import NoMatch from './NoMatch'
+import FoodDetails from './FoodDetails'
 
 export default function App () {
   return (
     <Router>
       <>
         <Route path='/' component={TopMenu} />
-        <Container style={{ paddingTop: 75 }}>
+        <Container style={{ marginTop: 75 }}>
           <Switch>
             <Route exact path='/' component={FoodList} />
             <Route component={NoMatch} />
+            <Route path='/details/:foodId' component={FoodDetails} />
```

Comment:
Please also add a link to each food that will take us to `/details/foodId`. Thanks.